uiowa/uiowa
# Expand run first list of websites

Resolves https://github.com/uiowa/uiowa/issues/5558

# To Test

- Checkout main, `ddev blt composer install` and run a `ddev blt dsa` on a couple websites that also include uiowa.edu, admissions, grad.admissions where they aren't first in the list.
- Run `ddev blt auda` and see that uiowa.edu is run first followed by whatever is next.
- Checkout `run_first_sites` and run `ddev blt auda` again. See that uiowa.edu admissions and grad.admissions in that order are run first.
